packages/devtools_app/lib/src/screens/inspector/diagnostics.dart

```diff
@@ -36,23 +36,23 @@ const bool _showRenderObjectPropertiesAsLinks = false;
 class DiagnosticsNodeDescription extends StatelessWidget {
   const DiagnosticsNodeDescription(
     this.diagnostic, {
-    this.isSelected,
+    this.isSelected = false,
     this.searchValue,
     this.errorText,
     this.multiline = false,
     this.style,
-    @required this.debuggerController,
+    required this.debuggerController,
     this.nodeDescriptionHighlightStyle,
   });
 
-  final RemoteDiagnosticsNode diagnostic;
+  final RemoteDiagnosticsNode? diagnostic;
```

Comment:
InspectorTreeController passes InspectorTreeNode.diagnostics here, that is nullable.